core/src/main/resources/hudson/model/Run/configure.jelly

```diff
@@ -29,11 +29,12 @@ THE SOFTWARE.
     <l:main-panel>
       <div class="behavior-loading"><l:spinner text="${%LOADING}"/></div>
       <f:form method="post" action="configSubmit" name="config">
+        <j:set var="readOnlyMode" value="${!h.hasPermission(it,it.UPDATE)}"/>
         <f:entry title="${%DisplayName}" help="/help/run-config/displayName.html">
           <f:textbox name="displayName" value="${it.hasCustomDisplayName()?it.displayName:''}"/>
         </f:entry>
         <f:entry title="${%Description}" help="/help/run-config/description.html">
-          <f:textarea name="description" value="${it.description}" codemirror-mode="${app.markupFormatter.codeMirrorMode}" codemirror-config="${app.markupFormatter.codeMirrorConfig}" previewEndpoint="/markupFormatter/previewDescription"/>
+          <f:textarea name="description" value="${it.description}" codemirror-mode="${app.markupFormatter.codeMirrorMode}" codemirror-config="${app.markupFormatter.codeMirrorConfig}" previewEndpoint="${h.hasPermission(it,it.UPDATE)?'/markupFormatter/previewDescription':null}"/>
```

Comment:
This change should be reverted.

While it looks off, it's consistent with job configuration forms and the global configuration, and should be addressed everywhere, ideally by changing `f:textarea`, instead of being a one-off change for this form field.

> <img width="1239" alt="Screenshot 2022-03-23 at 16 27 35" src="https://user-images.githubusercontent.com/1831569/159735703-f93c7193-445a-4730-9727-d005790a70fe.png">
> <img width="984" alt="Screenshot 2022-03-23 at 16 27 54" src="https://user-images.githubusercontent.com/1831569/159735749-f35f7662-6e5f-434f-b738-dca1443fa48a.png">

Reply:
I reverted it. Maybe we can address this by opening another issue

Reply:
We likely can address it later on and iterate across all previewEndpoint occurrences of `f:textarea`s.  I'm fine with dismissing my previous comment.